src/content/docs/apm/new-relic-apm/getting-started/introduction-apm.mdx

```diff
@@ -16,150 +16,171 @@ redirects:
   - /docs/apm
   - /docs/apm/new-relic-apm
   - /docs/apm 
+  - /docs/apm/new-relic-apm/getting-started/introduction-apm
 ---
 
-import apmScreenshot from 'images/apm-screenshot.png'
+import apmOverview from 'images/apm-intro-overview.png'
 
-Application performance monitoring (APM) is exactly what it sounds like: Monitoring of your web or non-web application's performance. APM supports apps for several programming languages, including [Go](/docs/agents/go-agent/get-started/introduction-new-relic-go), [Java](/docs/agents/java-agent/getting-started/introduction-new-relic-java), [.NET](/docs/agents/net-agent/getting-started/introduction-new-relic-net), [Node.js](/docs/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs), [PHP](/docs/agents/php-agent/getting-started/introduction-new-relic-php), [Python](/docs/agents/python-agent/getting-started/introduction-new-relic-python), and [Ruby](/docs/agents/ruby-agent/getting-started/introduction-new-relic-ruby), as well as a [C SDK](/docs/agents/c-sdk/get-started/introduction-c-sdk).
+import clogo from 'images/clogo.png'
+
+import gologo from 'images/gologo.png'
+
+import javalogo from 'images/javalogo.png'
+
+import dotnetlogo from 'images/dotnetlogo.png'
+
+import nodejslogo from 'images/nodejslogo.png'
+
+import pythonlogo from 'images/pythonlogo.png'
+
+import phplogo from 'images/phplogo.png'
+
+import rubylogo from 'images/rubylogo.png'
+
+New Relics' application performance monitoring (APM) provides a unified monitoring service for all your apps and microservices. Monitor everything from the hundreds of dependencies of a modern stack down to simple web-transaction times and throughput of an app. Keep track your app's health in real-time by monitoring your **metrics**, **events**, **logs**, and **transactions** (MELT) through pre-built and custom dashboards. 
+
+Our APM service provides the flexability to monitor the exact things you need from your app by automatically instrumenting your code when you install one of our agents. 
+
+- Is your early-access software still slightly unstable? Proactively monitor and solve those errors before they affect your users with errors inbox.  
+- What if your users always comment on how speedy your webapp is and you want to quantify that feedback? Measure their satisfaction by monitoring your Apdex in a glance.
+- Need somewhere to store your logs? Our agents automatically ingest them. 
+- What if your modern stack has dozens or even hundreds of dependencies to keep track of? Easily track them with automap and external services. 
+
+Want to save time with a single unified monitoring solution? Click a logo to get started with APM. It takes just a few minutes!
+
+
+<TechTileGrid>
```

Comment:
When I looked at this in Gatsby preview, the duplication of tiles really confused me. I don't care where you put them, but delete the other set. Otherwise it gives the impression that the two sets are for two different purposes.